_posts/apereo/2017-05-05-jotform-clas.md

```diff
@@ -0,0 +1,19 @@
+---
+layout:     post
+title:      Easier CLA submission
+summary:    A new online form supplements prior ways of submitting contributor license agreements.
+tags:       [Apereo, licensing]
+---
+
+Now it is easier to submit a contributor licensing agreement (CLA) to Apereo, with online forms, one for each type of agreement, linked from [the licensing page][1].
+
+Contributor agreements allow Apereo to sublicense your contributions to others under flexible terms, including under the open source licenses adopted by the Apereo projects. The Apereo Foundation collects and maintains licensing agreements and grant assignments at the institutional and individual level and [publishes the results publicly][2]. 
```

Comment:
To me this statement is overly technical and I anticipate a lot of readers will not know what sublicensing means.  I think the key point is the flexibility and ease of adoption and adaption.